pkg/kubelet/container/runtime.go

```diff
@@ -284,6 +284,24 @@ func (p *Pod) FindContainerByName(containerName string) *Container {
 	return nil
 }
 
+// ToAPIPod converts Pod to api.Pod. Note that if a field in api.Pod has no
+// corresponding field in Pod, the field would not be populated.
+func (p *Pod) ToAPIPod() *api.Pod {
+	var pod api.Pod
+	pod.UID = p.ID
```

Comment:
In the current stage, they are far from close. I am not sure if the plan is to make them close, or to grow them differently. @dchen1107 would know more...